3.10/arangosearch-case-sensitivity-and-diacritics.md

```diff
@@ -23,7 +25,16 @@ var analyzers = require("@arangodb/analyzers");
 analyzers.save("norm_en", "norm", { locale: "en.utf-8", accent: false, case: "lower" }, ["frequency", "norm", "position"]);
 ```
 
-**View definition:**
+### View definition
+
+#### Search Alias View
+
+```js
+db.imdb_vertices.ensureIndex({ type: "inverted", name: "inv-ci", fields: [ { name: "title", analyzer: "norm_en" } ] });
+db._createView("imdb_alias", "search-alias", { indexes: [ { collection: "imdb_vertices", index: "inv-ci" } ] });
+```
+
+#### ArangoSearch View
```

Comment:
Try tried to avoid "`arangosearch` Views" in headlines, also because of [Google's Style Guide](https://developers.google.com/style/headings#formatting-a-heading-or-title:~:text=Avoid%20using%20code%20items%20in%20headings%20when%20possible.%20If%20you%20must%20mention%20a%20code%20item%20in%20a%20heading%2C%20add%20a%20descriptive%20noun%20to%20the%20item%20in%20code%20font.%20For%20more%20information%2C%20see%20Grammatical%20treatment%20of%20code%20elements.)